pkgs/by-name/de/decasify/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+
+  # nativeBuildInputs
+  zstd,
+  pkg-config,
+  jq,
+  cargo,
+  rustc,
+  rustPlatform,
+
+  # buildInputs
+  typos,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "decasify";
+  version = "0.8.0";
+
+  src = fetchurl {
+    url = "https://github.com/alerque/decasify/releases/download/v${finalAttrs.version}/decasify-${finalAttrs.version}.tar.zst";
+    sha256 = "sha256-HTUAb/yL3H4B/n/Ecd/fDpnTYiqwco/E07sa6pFIIU4=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    nativeBuildInputs = [ zstd ];
+    dontConfigure = true;
+    hash = "sha256-GjSAdhR3TqyLP94YncFJ+XtTkrgItaupTUmIkkEZISA=";
+  };
+
+  nativeBuildInputs = [
+    zstd
+    pkg-config
+    jq
+    cargo
+    rustc
+    rustPlatform.cargoSetupHook
+  ];
+
+  buildInputs = [
+    typos
+  ];
+
```

Comment:
```suggestion
```
This seems to only be called by `git-cliff`, which shouldn't be happening when we're only building. If it is required though, it should be in `nativeBuildInputs` so that we can ensure the system building this can actually execute it

Reply:
Sorry this was leftover from work for a dev shell in the upstream flake, removing.